tests/sdk/local/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "1.0.0"
 edition = "2021"
 
 [dependencies]
-surrealdb = { path = "../../../sdk", features = ["kv-mem"] }
+surrealdb = { path = "../../../crates/sdk", features = ["kv-mem"] }
```

Comment:
Should we disable default features here? By default we enable the WS engine, so that will be active here too. It looks like the goal is to test only `kv-mem`.